lib/Runtime/PlatformAgnostic/ICU.h

```diff
@@ -8,8 +8,6 @@
 #ifdef WINDOWS10_ICU
 #include <icu.h>
 #else
-#define U_STATIC_IMPLEMENTATION 1
```

Comment:
Is this not still needed if building non-shared?

Reply:
It looks like Chakra.Build.props sets the define, but I don't know if you build with both static and dynamic linking of ICU or not..?

Reply:
We build with one or the other at any given time. So, the build files set this if ChakraICU=static now.